### PR TITLE
Add email address to the returned information, when using whoami(). Fix #73.

### LINKED ISSUE #73: can whoami return email address aswell?
It maybe be nice if the `resin whoami` returns `username<my_mail@email.com>` if the email is available, this just helps to better identify the account, especially if the user has more than 1 account.

### PR DESCRIPTION
**Pull request by mkorakakis**
[Issue #73](https://github.com/resin-io/resin-cli/issues/73).

### CODE REVIEW

File: lib/actions/auth.coffee [hunk 1]
```diff
@@ -177,5 +177,11 @@ exports.whoami =
 		resin.auth.whoami().then (username) ->
 			if not username?
 				throw new Error('Username not found')
-			console.log(username)
+			resin.auth.getEmail().then (email) ->
+				if not email?
+					throw new Error('Email not found')
+				console.log """
+					Username: #{username}
```
Comment (jviotti) [inline lib/actions/auth.coffee]: Can we use a vertical table here for consistency?

File: lib/actions/auth.coffee [hunk 2]
```diff
@@ -177,5 +178,15 @@ exports.whoami =
 		resin.auth.whoami().then (username) ->
 			if not username?
 				throw new Error('Username not found')
-			console.log(username)
+			resin.auth.getEmail().then (email) ->
+				if not email?
```
Comment (jviotti) [inline lib/actions/auth.coffee]: I don't think this check is necessary. I believe a user always has an email, and even if no, `visuals`'s table will add a blank space at the side of the EMAIL property, making it clear that there is no email.

Comment (Page-) [inline lib/actions/auth.coffee]: I think certain social accounts (twitter?) don't always have emails associated

Comment (jviotti) [inline lib/actions/auth.coffee]: @Page- True, but in that case the user would get:

```
== ACCOUNT INFORMATION
USERNAME: jviotti
EMAIL:                   
```

File: lib/actions/auth.coffee [hunk 3]
```diff
@@ -177,5 +178,15 @@ exports.whoami =
...
+					throw new Error('Email not found')
+				console.log visuals.table.vertical
+					username: username
```
Comment (jviotti) [inline lib/actions/auth.coffee]: You can use `{ username, email }` in this case.

File: lib/actions/auth.coffee [hunk 4]
```diff
@@ -177,5 +178,10 @@ exports.whoami =
 		resin.auth.whoami().then (username) ->
 			if not username?
 				throw new Error('Username not found')
-			console.log(username)
+			resin.auth.getEmail().then (email) ->
+				console.log visuals.table.vertical {username, email}, [
```
Comment (jviotti) [inline lib/actions/auth.coffee]: Please put spaces around the curly braces:

```coffee
{ username, email }
```

**Pull request merged**